buildingsync_asset_extractor/converter.py

```diff
@@ -44,172 +46,172 @@ def convert(value: float, original_type: PowerUnits, to: PowerUnits) -> float:
+        "ft-lbf/h": 59.99999999548061,
+        "ft-lbf/min": 1.0,
+        "Btu/s": 2.1417630158504952e-05,
+        "kBtu/hr": 7.710404773085137e-05,
+        "MMBtu/hr": 7.705565339097697e-08,
+        "therms/h": 7.710084732258458e-07,
     },
-    'Btu/s': {
-        'W': 1055.0637786054167,
-        'kW': 1.0550637786054167,
-        'hp': 1.414861628385436,
-        'MW': 0.0010550637786054166,
-        'Btu/hr': 3600.0270412846457,
-        'cal/h': 907191.5551218071,
-        'ft-lbf/h': 2801430.389424041,
-        'ft-lbf/min': 46690.506493917565,
-        'Btu/s': 1.0,
-        'kBtu/hr': 3600.0270412846457,
-        'MMBtu/hr': 0.003597767485044471,
-        'therms/h': 0.03599877612601682,
+    "Btu/s": {
+        "W": 1055.0637786054167,
+        "kW": 1.0550637786054167,
+        "hp": 1.414861628385436,
+        "MW": 0.0010550637786054166,
+        "Btu/hr": 3600.0270412846457,
+        "cal/h": 907191.5551218071,
+        "ft-lbf/h": 2801430.389424041,
+        "ft-lbf/min": 46690.506493917565,
+        "Btu/s": 1.0,
+        "kBtu/hr": 3.600027041284646,
+        "MMBtu/hr": 0.003597767485044471,
+        "therms/h": 0.03599877612601682,
     },
-    'kBtu/hr': {
-        'W': 0.2930710704408832,
-        'kW': 0.0002930710704408832,
-        'hp': 0.00039301416688263316,
-        'MW': 2.930710704408832e-07,
-        'Btu/hr': 1.0,
-        'cal/h': 251.99576134241534,
-        'ft-lbf/h': 778.1692629798606,
-        'ft-lbf/min': 12.969487717307913,
-        'Btu/s': 0.0002777756912745735,
-        'kBtu/hr': 1.0,
-        'MMBtu/hr': 9.993723502034118e-07,
-        'therms/h': 9.999584923442933e-06,
+    "kBtu/hr": {
+        "W": 293.0710704408832,
+        "kW": 0.2930710704408832,
+        "hp": 0.3930141668826332,
+        "MW": 0.00029307107044088316,
+        "Btu/hr": 1000.0,
+        "cal/h": 251995.76134241532,
+        "ft-lbf/h": 778169.2629798605,
+        "ft-lbf/min": 12969.487717307913,
+        "Btu/s": 0.27777569127457347,
+        "kBtu/hr": 1.0,
+        "MMBtu/hr": 0.0009993723502034117,
+        "therms/h": 0.009999584923442934,
     },
-    'MMBtu/hr': {
-        'W': 293255.1319648094,
-        'kW': 293.2551319648094,
-        'hp': 393.26099706744867,
-        'MW': 0.29325513196480935,
-        'Btu/hr': 1000628.0439882698,
-        'cal/h': 252154025.76539588,
-        'ft-lbf/h': 778657987.5073314,
-        'ft-lbf/min': 12977633.126099706,
-        'Btu/s': 277.95014662756597,
-        'kBtu/hr': 1000628.0439882698,
-        'MMBtu/hr': 1.0,
-        'therms/h': 10.005865102639294,
+    "MMBtu/hr": {
+        "W": 293255.1319648094,
+        "kW": 293.2551319648094,
+        "hp": 393.26099706744867,
+        "MW": 0.29325513196480935,
+        "Btu/hr": 1000628.0439882698,
+        "cal/h": 252154025.76539588,
+        "ft-lbf/h": 778657987.5073314,
+        "ft-lbf/min": 12977633.126099706,
+        "Btu/s": 277.95014662756597,
+        "kBtu/hr": 1000.6280439882697,
```

Comment:
```suggestion
        "kBtu/hr": 1000.0,
```
Can we hard code these absolute conversion? It seems the precision setting introduces some tails